src/main/java/net/milkbowl/vault/economy/Economy.java

```diff
@@ -80,7 +81,7 @@
 
     /**
      * 
-     * @deprecated As of VaultAPI 1.4 use {@link #hasAccount(OfflinePlayer)} instead.
+     * @deprecated As of VaultAPI 1.4 use {@link #hasAccount(OfflinePlayer)} or {@link #hasAccount(UUID)} instead.
      */
     @Deprecated
     public boolean hasAccount(String playerName);
```

Comment:
I think we should remove these deprecated methods from the API. Vault 1.4 was out ages ago.

Reply:
I'll got over the review in a bit, but we cannot remove those as they are still widely used (by Towny and Factions and anyone using non-player accounts.)

Reply:
These will have to remain deprecated for the time being. Many plugins still expect them to be present. I'd say 6 months after an update that includes the UUID methods will be fair warning.